cloudinit/distros/freebsd.py

```diff
@@ -155,4 +171,9 @@ def update_package_sources(self):
             "update-sources", self.package_command,
             ["update"], freq=PER_INSTANCE)
 
+
+class Distro(FreeBSD):
```

Comment:
If we leave the other class named `Distro`, this I believe we can remove this class.